src/app/csc/csc.component.html

```diff
@@ -3,46 +3,41 @@
   <h2>{{ title }} Projects</h2>
   <div class="row csc">
     <div class="col-md-2">
+      <!-- Filter Section -->
       <div class="select-box">
         <h3 class="select-title">Topic</h3>
         <div *ngFor="let topic of topics">
-          <input
-            type="checkbox"
+          <mat-checkbox
```

Comment:
These need to be formatted to look like the current production site

Reply:
Done!